docs/getting-started/docker.md

```diff
@@ -59,6 +59,33 @@ Build the Docker image locally. The frontend will build followed by the backend.
 docker build -t clutch:latest .
 ```
 
+:::info
+If you encounter this error:
+```
+ => ERROR [gobuild 7/7] RUN make backend-with-assets                                                                            0.3s
+------
+ > [gobuild 7/7] RUN make backend-with-assets:
+0.196 /go/src/github.com/lyft/clutch
+0.196 Running pre-flight checks...
+0.212 Pre-flight checks satisfied!
+0.213 cd backend && go run cmd/assets/generate.go ../frontend/packages/app/build && go build -tags withAssets -o ../build/clutch -ldflags="-X main.version=0.0.0"
+0.218 go: errors parsing go.mod:
+0.218 /go/src/github.com/lyft/clutch/backend/go.mod:3: invalid go version '1.23.0': must match format 1.23
```

Comment:
Seems to me we could do with just the error message instead of the log snippet for readability. And then do something like `If the make backend build fails with ERROR_MSG, modify the golang version...`